Index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  <div class="container">
+    <nav class="navbar">
+      <ul class="nav-list header-menu">
+        <li ><a href="Index.html">Home</a></li>
+        <li ><a href="/About.html">About</a></li>
+      </ul>
+    </nav>
+           <div class="contents">
+                    <h2 class="contents-head">"Hello! Sharing World"</h2>
+                    <h1 class="contain">GREATIVE COMMONS<BR>GLOBAL SUMMIT<BR>2015</h1>
+                      <div class="load">
+                        <P class="paragraph">
+                            A joyful celebration in the value of openness and  sharing, ceating <br> a positive change with people from over 80 countries is taking place, in october Korea
+                        </P>
+                        <h3 class="date">2015.10.15(THU) ~ 16(FRI)</h3>
+                        <p >@ National Museam of Korea, Art Center Nabi and more</p>
+                      </div>
+                </div>
```

Comment:
Kindly ensure that the hero section is as close to the design as possible.

Points to note:

- Adjust the font family, size and colors to match
- Customize the content while maintaining the design
- The background is not purely white. It has a background image with a white overlay

Your Design:
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/324152ce-bc21-4402-b792-52e758b00013)

Design Guide:
![image](https://github.com/maserekafelex/Capston-project/assets/20554583/a9d4326d-c6b4-465a-9e1c-39fcb9407fb4)